src/resources/postcss/views/day/_event.pcss

```diff
@@ -0,0 +1,86 @@
+.tribe-events {
+
+	/* -----------------------------------------------------------------------------
+	 * Event
+	 * ----------------------------------------------------------------------------- */
+
+	.tribe-events-calendar-day__event {
+		display: flex;
+		flex-direction: column;
+		margin-bottom: var(--spacer-5);
+		margin-top: var(--spacer-8);
+
+		@media (--viewport-medium) {
+			flex-direction: row-reverse;
+			justify-content: flex-end;
+			margin-bottom: var(--spacer-7);
+			margin-top: var(--spacer-10);
+		}
+
+		&:last-child {
+			margin-bottom: var(--spacer-8);
+
+			@media (--viewport-medium) {
+				margin-bottom: var(--spacer-10);
+			}
+		}
+	}
+
```

Comment:
also missing the case for `.tribe-events-calendar-day__time-separator + .tribe-events-calendar-day__event`, see https://github.com/moderntribe/the-events-calendar/pull/2583/files#diff-ff3de376ecb4382618c6419ad7a5b4f4R25